django-stubs/contrib/auth/backends.pyi

```diff
@@ -1,19 +1,16 @@
-from typing import Any, TypeAlias, TypeVar
+from typing import Any
 
-from django.contrib.auth.base_user import AbstractBaseUser, _UserModel
-from django.contrib.auth.models import AnonymousUser, Permission
+from django.contrib.auth.models import Permission, _AnyUser, _User, _UserType
+from django.contrib.auth.models import _UserModel as UserModel  # noqa: F401
```

Comment:
Oh, this is not right. Let's use `from django.contrib.auth.models import _UserModel; UserModel = _UserModel`

Because `as` should have the same name for mypy. It is weird, I know :)

Reply:
I didn't think mypy was that picky about the names, but it appears it is, and probably why ruff has the same logic